.gitlab-ci.yml

```diff
@@ -101,7 +101,7 @@ build:debug:
   - export LC_ALL=C.UTF-8
   - export LANG=C.UTF-8
   - |
-    if ! make -C vpp -j8 ${TEST_TARGET} TEST=test_upf V=1; then
+    if ! make -C vpp -j8 ${TEST_TARGET} TEST=test_upf V=1 EXTERN_TESTS=../../upf/test;; then
```

Comment:
doesn't that belong into e2e PR?

Reply:
That's restoring pythonic Scapy test that stopped being run due to repo relayout, not e2e. The test in question was affected by the problem this PR fixes. I wouldn't mind it being split into another commit though (not sure it deserves a separate PR ...)